src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -340,18 +377,28 @@ public void beforeClickOn(WebElement element, WebDriver driver) {
 
   @Override
   public void afterClickOn(WebElement element, WebDriver driver) {
-    StringBuilder builder = new StringBuilder();
-    builder.append("<tr class=\"success lowLevelAction\"><td>click</td><td>" + lastFindBy
-        + "</td><td> <br/> &nbsp;</td></tr>");
-    CommonUtils.appendTextToFile(logPath, builder.toString());
+    List<String> classList = new ArrayList<>();
+    classList.add(SUCCESS_CLASS);
+    classList.add(LOW_LEVEL_ACTION_CLASS);
+    String command = "click";
+    //TODO:Check toString()
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)